src/service/webviewService.ts

```diff
@@ -0,0 +1,39 @@
+import * as vscode from "vscode";
+import CONSTANTS from "../constants";
+import * as path from "path";
+import { GETTING_STARTED_HTML } from "../pages/gettingStarted";
+
+// Manages different type of webview
+export class WebviewService {
```

Comment:
Is there a plan to port the CPX/Microbit opening logic here as well? If not, it doesn't make much sense to call it webviewService as it only managing your tutorials page

Reply:
Yes, it will be used in the future to manage all different webviews including the release notes, microbit, cpx as well.